x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/user/InternalUsers.java

```diff
@@ -144,7 +146,9 @@ public class InternalUsers {
                         ForceMergeAction.NAME + "*",
                         // indices stats is used by rollover, so we need to grant it here
                         IndicesStatsAction.NAME + "*",
-                        UpdateSettingsAction.NAME
+                        UpdateSettingsAction.NAME,
```

Comment:
Non-blocking: let's add these to the `InternalUsersTests::testDataStreamLifecycleUser` -- we want to include all new actions in `sampleIndexActions` and adjust the assertion for non-system data streams and tweak ` allowedSystemDataStreams.forEach(allowedSystemDataStream {}` to assert allow/deny based on the which action we're dealing with.